website/docs/components/table/advanced-table/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,243 @@
+
+![Advanced Table sticky header example](/assets/components/table/advanced-table/advanced-table-sticky-header.png)  
+
+## Cells
+
+### Density
+
+![Table cell density](/assets/components/table/advanced-table/table-density.png)
+
+- We recommend using medium cell density by default.
+- If content is largely string/text-based, short allows for more content to be displayed within the page.
+- While denser content allows for more rows to be displayed within a single page, it also makes comprehension and scanning more difficult.
+
+### Column/Row Span
+
+- Supports merging multiple columns or rows.
+- Use column and row spans carefully to avoid misalignment or breaking table interactions.
+- Multi-span cells should use the same alignment for readability.
+
+![Table cell density](/assets/components/table/advanced-table/colspan-table-example.png)
+
+### Horizontal scrolling
+
+- Needed when there are many columns of equal priority.
+- Place the Advanced Table in a scrollable container for smooth side-scrolling.
+
+![Advanced Table horizontal scrolling](/assets/components/table/advanced-table/horizontal-scrolling.png)
+
+## Keyboard navigation
+
+There are two modes when using a keyboard to interact with the Advanced Table: navigation and action.
+
+### Navigation mode
+
+When the user moves focus into the Advanced Table (tab key), the first cell will be focused, putting them into navigation mode and allowing the user to navigate freely with their arrow keys. Other keys within navigation mode include: 
+
+- Home (fn + left): Move focus to the row's first cell.
+- End (fn + right): Move focus to the row's last cell.
+- PageUp (fn + up): Moves focus to the first cell in the column.
+- PageDown (fn + down): Moves focus to the last cell in the column.
+- Horizontal Scrolling: Arrow keys allow horizontal movement to access hidden columns if the Table extends beyond the screen width.
+
+### Action mode
+
+When the user is in navigation mode, pressing enter will activate action mode, where:
+
+- If a cell contains a single interactive element, pressing Enter will focus on that element instead of executing it immediately.
+- If there is more than one interactive element within the cell, the first element will be focused, allowing the user to tab between them.
+- Pressing escape will take the user out of action mode and back into navigation mode.
+
+### Multi-Select
+
+Multi-select allows users to select multiple rows to perform bulk actions, such as deleting or exporting data. The Advanced Table maintains selection states across pagination and filtering, ensuring consistency when interacting with large datasets. However, multi-select and sorting are not supported for nested rows at this time. For more details, check out the [Multi-Select Table Pattern.](https://helios.hashicorp.design/patterns/table-multi-select)
```

Comment:
```suggestion
Multi-select allows users to select multiple rows to perform bulk actions, such as deleting or exporting data. Multi-select and sorting are not supported for nested rows at this time. For more details, check out the [Multi-Select Table Pattern.](https://helios.hashicorp.design/patterns/table-multi-select)
```

I don't believe the Advanced Table maintains the selection state across pagination (that is not the case with the original Table component), I think the consumer has to do that. Is this correct @alex-ju?